test/e2e-tests-yaml.sh

```diff
@@ -33,12 +33,7 @@ install_pipeline_crd
 
 # Run the tests
 failed=0
-# FIXME(vdemeester) add pipelinerun to the test…
-# They are currently block by
-# - https://github.com/knative/build-pipeline/issues/375
-# - https://github.com/knative/build-pipeline/pull/443
-# for test in taskrun pipelinerun; do
-for test in taskrun; do
+for test in taskrun pipelinerun; do
```

Comment:
@bobcatfish trying something :upside_down_face: related to #477

Reply:
:sweat_smile: